paper/paper.bib

```diff
@@ -0,0 +1,184 @@
+@article{ac2000,
+  author = {A Agresti and B Caffo},
+  year = {2000},
+  title = {Simple and Effective Confidence Intervals for Proportions and Differences of Proportions Result from Adding Two Successes and Two Failures},
+  journal = {The Americal Statistician},
+  volume = {54},
+  number = {4},
+  pages = {280-288},
+  doi = {10.2307/2685779}
+}
+
+@article{barker2002,
+  author = {L Barker},
+  title = {A Comparison of Nine Confidence Intervals for a Poisson Parameter When the Expected Number of Events is ≤ 5},
+  journal = {The Americal Statistician},
+  volume = {56},
+  number = {2},
+  pages = {85-89},
+  year = {2002},
+  doi = {10.1198/000313002317572736}
+  }
+
+  @article{ba1986,
+  author = {J M Bland and D G Altman},
+  title = {Statistical methods for assessing agreement between two methods of clinical measurement },
```

Comment:
The titles are not consistent, capital letters.

Reply:
bibtex will handle that